src/components/filterPanel/FilterPanel.js

```diff
@@ -25,24 +39,61 @@ function getStatusClassName(status) {
   return statusClassMapper[status];
 }
 
+const exportUrl = `${endpoints.blackspotsExport}`;
+
+export const getExportFilter = stadsdeelFilter => {
+  if (Object.values(stadsdeelFilter).filter(e => e).length === 0) return '';
+  const stadsdeel = Object.keys(stadsdeelFilter).find(
+    key => stadsdeelFilter[key] === true
+  );
+  const filterValue = Object.values(Stadsdeel).find(
+    ([key, value]) => value.name === stadsdeel
+  );
+  return `stadsdeel=${filterValue[1].value}`;
+};
+
 const FilterPanel = ({
   spotTypeFilter,
   spotStatusTypeFilter,
   blackspotYearFilter,
   deliveredYearFilter,
   quickscanYearFilter,
+  stadsdeelFilter,
   setFilters,
   setBlackspotListFilter,
   setQuickscanListFilter,
   setDeliveredListFilter,
+  setStadsdeelFilter,
 }) => {
   const [optionValue, setOptionValue] = useState(ContextMenuOptions.ALL);
   const [showPanel, setShowPanel] = useState(true);
   const { trackEvent } = useMatomo();
+  const [downloadUrl, setDownloadUrl] = useState(exportUrl);
+  const [canDownload, setCanDownload] = useState(true);
 
-  const trackFilter= (name) => {
-    trackEvent({category: 'Map filters', action: name});
-  }
+  const [, downloadFile] = useDownload();
+
+  const exportFilter = () => {
+    downloadFile(
+      downloadUrl,
+      `wbakaart-export-${(new Date()).toLocaleDateString("nl-NL")}.csv`
+    );
+  };
+
+  useEffect(() => {
+    setDownloadUrl(`${exportUrl}${getExportFilter(stadsdeelFilter)}`);
+    setCanDownload(
+      Object.values(stadsdeelFilter).filter(e => e).length <= 1 &&
+        Object.values(spotTypeFilter).filter(e => e).length === 0 &&
+        Object.values(spotStatusTypeFilter).filter(e => e).length === 0 &&
+        optionValue === ContextMenuOptions.ALL
+    );
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [stadsdeelFilter]);
```

Comment:
Dependencies should include `spotTypeFilter` and `spotStatusTypeFilter`

Reply:
indeed, missed that!